tests/testthat/_snaps/selections.md

```diff
@@ -0,0 +1,25 @@
+# simple name selections
+
+    Code
+      recipes_eval_select(quos = quos(log(date)), data = okc, info = info1)
+    Error <rlang_error>
+      non-numeric argument to mathematical function
+      Caused by error in `log()`:
+      ! non-numeric argument to mathematical function
```

Comment:
this double showing of the error is strange, i dont think it is anything youve done though

Reply:
Perfect, thank you!